### PR TITLE
Add isMerchantInitiated

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 1]
```diff
@@ -131,6 +131,10 @@ properties:
   3ds:
     allOf:
       - $ref: ../Transactions/ThreeDSecureResult.yaml
+  isProcessedOutside:
+    type: boolean
+  isMerchantInitiated:
+    type: boolean
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonTransaction.yaml:137]: Why did you add i CommonTransaction?

Comment (ashkarpetin) [reply openapi/components/schemas/Common/CommonTransaction.yaml:137]: Common transaction is used by StorefrontTransaction schema and by Transaction schema

Comment (colinbird) [reply openapi/components/schemas/Common/CommonTransaction.yaml:137]: Ok, thanks, will fix

File: openapi/components/schemas/Common/CommonTransactionRequest.yaml [hunk 2]
```diff
@@ -93,6 +93,10 @@ properties:
     description: True if transaction was processed outside Rebilly.
     type: boolean
     default: false
+  isMerchantInitiated:
+    description: True if transaction was initiated by the merchant.
```
Comment (colinbird) [inline openapi/components/schemas/Common/CommonTransactionRequest.yaml:97]:
```suggestion
    description: True if the transaction was initiated by the merchant.
```

**Review by ashkarpetin — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]
